autoscale-api/src/main/resources/messages/messages.properties

```diff
@@ -3,6 +3,7 @@ autoscale.unsupported.hostgroups=Unsupported HostGroup ''{0}'' for {1} Autoscali
 
 autoscale.config.not.found=''{0}'' Autoscale Configuration with identifier ''{1}'' not found for Cluster ''{2}''.
 autoscale.entitlement.not.enabled=Autoscaling Entitlement is not enabled for CloudPlatform ''{0}'', Cluster ''{1}''.
+autoscale.stopstart.entitlement.not.enabled=StopStart scaling mechanism entitlement is not enabled for CloudPlatform ''{0}'', Cluster ''{1}''.
```

Comment:
Maybe `Stop/start scaling...` would be more friendly.

Reply:
Ack. Will change the messages.

Reply:
Fixed in the next push. Resolving conversation.